#name : try block : ...
# ---
try:
    $0
except Exception:
    print(sys.exc_info() )